.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ initialize-venv: &initialize-venv
 publish-prerelease-on-github: &publish-prerelease-on-github
   name: Publish wheel and source distribution on github
   command: |
-    if [ "${CIRCLE_BRANCH}" == "dev" ]; then
+    if [[ "${CIRCLE_BRANCH}" == "dev" && "${PUBLISH_RELEASE}" == "true" ]]; then
```

Comment:
This should always be done.

Reply:
The variable should probably be named  `UPLOAD_WHEELS` or `PUBLISH_WHEELS` ... having the word `RELEASE` in the name of the variable seems that the actual "stable release" is published ...

Reply:
Yup I agree; this tricked me until you made this comment.

Reply:
Ok, I rebased it (changes in the cirlcle config are gone for this PR) and will merge this since this PR fixed the travis issue.

Reply:
Full disclosure, I made the commit adding this variable. I will do a follow up Pr as soon as CI fixes are all integrated